src/security/ZeroSensitiveMemory.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 1996-2021 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_SRC_SECURITY_ZEROSENSITIVEMEMORY_H
+#define SQUID_SRC_SECURITY_ZEROSENSITIVEMEMORY_H
+
+#include <cstring>
+
+namespace Security {
+
+inline void
+ZeroSensitiveMemory(void *dst, const size_t len)
```

Comment:
I suggest moving the new function to src/mem/Sensitive.h. That move cannot cause any new linking problems AFAICT because the function will continue to be inlined. I do not insist on this move if others prefer the current src/security/ZeroSensitiveMemory.h location.